deploy/infrastructure/main.tf

```diff
@@ -3,7 +3,7 @@ locals {
 
   {%- if app_url_type == "subdomain" %}
 
-  app_url = join(".", compact([var.app_subdomain, {% if dns_provider == "aws" %}data.aws_route53_zone.this.name{% elif dns_provider == "cloudflare" %}data.cloudflare_zone.this.name{% endif %}]))
+  app_url = join(".", compact([var.app_subdomain, var.zone_name]))
```

Comment:
If I am not mistaken, in both cases we end up with the zone name, therefore I simplified

Reply:
I think its fine to be a little more static in the way we set these base values